tests/python/rest_api/test_tasks.py

```diff
@@ -590,21 +590,30 @@ def test_can_split_skeleton_tracks_on_jobs(self, jobs):
 
 @pytest.mark.usefixtures("restore_db_per_class")
 class TestGetTaskDataset:
-    def _test_export_task(self, username, tid, **kwargs):
+    DATASET_FORMAT = "CVAT for images 1.1"
+
+    def _test_export_task(
+        self, username: str, tid: int, dataset_format: Optional[str] = None, **kwargs
```

Comment:
```suggestion
        self, username: str, tid: int, *, format: Optional[str] = None, **kwargs
```

There doesn't seem to be much reason to change `format` to `dataset_format`.